queries/javascript.core.scm

```diff
@@ -370,6 +370,14 @@
   (_) @value
 ) @_.domain
 
+;;!! str => str.length > 0
+;;!         ^^^^^^^^^^^^^^
+;;!  ---------------------
+(arrow_function
+  body: (_) @value
+  (#not-type? @value statement_block)
```

Comment:
I agree this is prob the right place to start. We'll see if we end up wanting it for blocks as well in the future

Reply:
That should probably be the `body` scope instead?